### ISSUE TITLE
API Gateway

### ISSUE THREAD

**Issue by tanandy** [ovh/public-cloud-roadmap#91]
As a user I want to be able to use create/update/delete API Gateway for the public cloud
I want to be able to consume these features through standard APIs / OVHcloud Customer panel
So that I i will be able to create, publish, maintain, monitor, and secure APIs at any scale

**Comment by mhurtrel** [comment]
Hi @tanandy Though I completely agree with the interest in such technology, we dont see yet a lot of traction for this need. We also hear many customers relying on managed service meshes on top of kubernetes to asnwer the same use cases, which we are exploring, with for example a istio add-on to the managed kubernetes service. Would that suit your needs ? If not, do you have a reversible and stable tech in mind ?

**Comment by tanandy** [comment]
Hi, as you know Service mesh is a bit different from an API Gateway one of the main thing for example is that Service mesh will interact directly at internal level that is different from Api gateway. you have good article talking about it here : 
https://konghq.com/blog/the-difference-between-api-gateways-and-service-mesh/ 

For technology i think OVH will need to write its own API gateway to get something really valuable that will be able to interact to its others services like IAM etc...

thats the way different cloud providers have choosen and also make some partnership if they were not ready like apigee but i really prefer something fully integrated into the OVH env.

**Comment by mhurtrel** [comment]
I agree with your augments and leave the issue open for comments/suggestions.
I do not yet make it part of the backlog yet as we have limited resources and many other issues prioritized for the next 12 months. We will reconsider the feature on a regular basis, and look at the feedback here.

**Comment by tanandy** [comment]
ok i hope OVH will recruit soon (#IPO)

**Comment by mhurtrel** [comment]
Hello, juste an informal update to confirm we still consider this aligned with our mid/long term strategy but have no formal plan to release this in the upcoming year. We will update this as we come closer to releasing an API gateway product. Feel free to keep commenting/giving precisions about your requirements.

**Comment by jylenhof** [comment]
Since last status was more than one year ago, any update ?
